### PR TITLE
#180 - update license

### LINKED ISSUE #180: Change LICENSE to MIT

### PR DESCRIPTION
**Pull request by rohrschacht**
Changes the license as requested. Closes #180.

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# CSIRO Open Source Software Licence Agreement (variation of the BSD / MIT License)
+# Ethereum Logging Framework
+
+The Blockchain Logging Framework is based on the [Ethereum Logging Framework](https://github.com/ChrisKlinkmueller/Ethereum-Logging-Framework),
+which is licensed under the CSIRO Open Source Software Licence Agreement. All code
+that was inherited from the Ethereum Logging Framework is therefore still licensed
+under the CSIRO Open Source Software Licence Agreement as stated below.
+
+## CSIRO Open Source Software Licence Agreement (variation of the BSD / MIT License)
```
Comment (daniil-romanchenko) [inline NOTICE.md:8]: Nice idea to also include this 👍

**Review by daniil-romanchenko — APPROVED** [review]
Looks all good to me!

File: NOTICE.md (hunk 1 above)
Comment (rohrschacht) [reply NOTICE.md:8]: Thanks!

**Review by Knostpe — APPROVED** [review]
Looks great, thanks for the changes guys!

**Pull request merged**